src/main/java/com/hillel/items_exchange/exception/InvalidCategoryVoException.java

```diff
@@ -0,0 +1,11 @@
+package com.hillel.items_exchange.exception;
+
+import lombok.Getter;
+
+@Getter
+public class InvalidCategoryVoException extends RuntimeException {
```

Comment:
I don't see any difference between `InvalidCategoryVoException` and `InvalidSubcategoryVoException`. You could simplify it to one exception with parameter.